src/app/components/normal-component/pick-game/pick-game.component.ts

```diff
@@ -214,6 +217,8 @@ class GameDescription {
 
     public static readonly QUARTO: Localized = () => $localize`Create a winning alignment. The problem: you don't pick the piece that you're placing on the board!`;
 
+    public static readonly QUEBEC_CASTLES: Localized = () => $localize`Everyboard first game: one player attack, the other defend, none have similar pieces!`;
```

Comment:
`EveryBoard's first game: one player attacks, the other defends. They each move in different ways!` peut-être ? (car leur pièces sont similaires pour un certain degré de similarité !)

Reply:
Euh, je trouves pas ça super comme description :D

Reply:
brainstormons, car "un joueur attaque, l'autre défend" c'est pas top non plus !

Reply:
ou alors je me rerend à coder et recrée une autre config ou il n'y a plus de camps envahisseur et le défenseur est au centre et sa victoire arrive après X tours haha, ce sera plus facile que trouver une phrase catch :D

Reply:
haha :D Bah j'aime bien les camps dans les coins, et la dynamique asymétrique. C'est juste qu'on est pas défenseur/attaquant. 
> EveryBoard's first game: an asymmetric game where each player moves differently. Be the first one to capture the opponent's throne

Par exemple ?

Reply:
On a dit throne mais on pourrait dire castle pour rester dans le thème du jeu (même si un castle contient un throne), je te laisse choisir et résoudre !